core/domain/question_domain_test.py

```diff
@@ -366,6 +368,18 @@ def test_strict_validation_for_answer_groups(self):
         self._assert_validation_error(
             'Expected all answer groups to have destination as None.')
 
+    def test_validation_for_inapplicable_misconception_ids(self):
+        """Test to verify that the validation fails when
+        inapplicable_misconception_ids field is invalid.
+        """
+        self.question.inapplicable_misconception_ids = ['Test', 1]
```

Comment:
Should we also add test for `inapplicable_misconception_ids` != list*

Reply:
Done.